app/config/tables/eonasdan/forms/eonasdan/formDef.json

```diff
@@ -1,8 +1,58 @@
 {
   "xlsx": {
+    "Export Summary": [
```

Comment:
Looks like an extra sheet was added here.

Reply:
Whoops, using Mac numbers this accidentally happened. Removed